packages/dci-api/src/async-search/async-search-handler.test.ts

```diff
@@ -21,10 +22,16 @@ describe('POST /registry/search', () => {
   beforeEach(async () => {
     const { init } = await createServer()
     server = await init()
+    mock.method(
+      sign,
+      'generateSignature',
+      async () => await Promise.resolve('')
+    )
```

Comment:
Could not get the test to pass without increasing the setTimeout interval. Just mocked away the function instead

Reply:
should be fine, maybe later we can write an actual test to the signature that checks it matches the message properly